### DISCUSSION TITLE
Next.js CSS not respecting px, rem, vw when inspecting mobile on Chrome / Firefox

### DISCUSSION THREAD

**Discussion by rptrainor** [vercel/next.js#13558] [Help] [answered]
Hello. I am working with Next.js for the first time and using a layout component, _app.js, global CSS, and layout.modules.css, but pixels and the viewport width is not the same from desktop to simulated mobile views in both Chrome and Firefox.  

For example in the below screenshots I am trying to center the profile image, but `html { min-width: 100%; }` and `html { min-width: 100vw; }` works as expected on the normal desktop view in the browser, but when you simulate mobile screens 100vw !== the full width of the screen.  And even on mobile 1000px !== 1000px on desktop. 

This might be a noob question, but I would really appreciate if you would help me figure this one out.  I have been really excited about using Next

![Screen Shot 2020-05-29 at 9 17 07 AM](https://user-images.githubusercontent.com/12374439/83282436-6f1fbb80-a18e-11ea-884d-b382cb4faa01.png)
![Screen Shot 2020-05-29 at 9 18 01 AM](https://user-images.githubusercontent.com/12374439/83282456-76df6000-a18e-11ea-8f5e-488e2bddb126.png)
![Screen Shot 2020-05-29 at 9 20 04 AM](https://user-images.githubusercontent.com/12374439/83282483-82328b80-a18e-11ea-963c-3b9c49d19b1c.png)

**Comment by jamesmosier** [comment, marked as answer]
Hi @rptrainor. Could you perhaps share the repository or a reproducible code sample? Or even maybe even just minimally the `layout.module.css` file would help 🤔

**Reply by rptrainor** [reply]
Thanks for taking a look at it.  The repo is here [https://github.com/rptrainor/ryanptrainor](https://github.com/rptrainor/ryanptrainor) 

Yes, just trying to get `html` or `body` to span the full window on mobile screens the same way it does with desktop screens.  Reproduced in both Chrome and Firefox.  And the exact same code works as expected with Gatsby, so I think it is something with my Next config.

**Reply by jamesmosier** [reply]
I was able to resolve it by adding this CSS:

```css
img {
  max-width: 100%;
  height: auto;
}
```

Previously the images were full size and causing rendering issues with the rest of the page. That CSS will size the images to fit within the viewport.

**Reply by rptrainor** [reply]
Thank you, I appreciate your help.  It was what I suspected - a noob question.  But I learned something today.

**Reply by jamesmosier** [reply]
Glad to hear it 🚀  and believe me, there are no noob questions when it comes to this stuff 😄

**Reply by ghost** [reply]
I have the same issue. My width is decreasing on mobile phones
